lib/createValidateDirectiveVisitor.test.ts

```diff
@@ -18,32 +18,35 @@ describe('createValidateDirectiveVisitor', (): void => {
   });
 
   it('defaults work', async (): Promise<void> => {
-    const directive = createValidateDirectiveVisitor({
+    const Directive = createValidateDirectiveVisitor({
```

Comment:
nit: why rename the variable ? Isn't this the `Visitor`'s instance ?

Reply:
no, `createValidateDirectiveVisitor` returns a class 

https://github.com/profusion/apollo-validation-directives/blob/118e75b75af632085bbe94091c398af652a262f9/lib/createValidateDirectiveVisitor.ts#L87